.circleci/config.yml

```diff
@@ -624,14 +624,14 @@ jobs:
     resource_class: xlarge
     environment:
       TAG: circleci-nightly
+      VERSION: 0.8-dev
```

Comment:
which pr/build made this version/what's different about it ?

Reply:
Default is 0.5.0 
Circle is building nightly versions of 0.8 - obviously 0.8.0. would be a bad name. 
Not sure what is the problem here - continuing to generate '0.5.0' doesn't seem right.